src/test/java/org/broadinstitute/consent/http/service/DatasetServiceTest.java

```diff
@@ -767,10 +768,46 @@ void testApproveDataset_DenyDataset() throws Exception {
     verify(emailService, times(1)).sendDatasetDeniedMessage(
         user,
         "DAC NAME",
-        "DUOS-000001"
+        "DUOS-000001",
+        "dacEmail@gmail.com"
     );
   }
 
+  @Test
+  void testApproveDataset_DenyDataset_WithNoDACEmail() throws Exception {
+    Dataset dataset = new Dataset();
+    dataset.setDataSetId(1);
+    User user = new User();
+    user.setUserId(1);
+    user.setEmail("asdf@gmail.com");
+    user.setDisplayName("John Doe");
```

Comment:
`User` has a constructor that can be used here
```suggestion
    User user = new User(1, "asdf@gmail.com, "John Doe", null);
```